app/assets/src/components/ui/controls/dropdowns/common/DropdownTrigger.jsx

```diff
@@ -29,9 +77,27 @@ class DropdownTrigger extends React.Component {
         )}
         onClick={onClick}
       >
-        <div className={cs.labelContainer}>
-          {label && <span className={cs.label}>{label}</span>}
-          <span className={cx(value === null && cs.placeholder)}>
+        <div
+          className={cs.labelContainer}
+          ref={hideBadgeIfInsufficientSpace && this.labelContainerRef}
+        >
+          {label && (
+            <span
+              className={cx(
+                cs.label,
+                hideBadgeIfInsufficientSpace && cs.disableMarginRight
```

Comment:
Do you need disableMarginRight? It seems like the behavior is the same whether you disable it or not.

Reply:
Yes I need disableMarginRight because otherwise the margin messes up `const labelTextWidth = this.labelRef.current.offsetWidth;` and I would have to add in `-3.7` again